### PR TITLE
Android Payment Apps developer's guide update

### PR DESCRIPTION
**Pull request by agektmr**
What's changed, or what was fixed?
- Add a sentence to the Android Payment Apps developer's guide

- [ ] This has been reviewed and approved by ( @sahel-sh )
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele @sahel-sh

### CODE REVIEW

File: src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md [hunk 1]
```diff
@@ -289,6 +289,16 @@ choose a different payment app. The merchant website does not observe this, so
...
 
+Note: In Chrome 84 or later, if the activity result of a payment response
+received from the invoked payment app is set to `RESULT_OK`, then Chrome will
+check for non-empty `methodName` and `details` in its extras. If the validation
+fails Chrome will reject the `request.show()` promise with one of the following
+developer facing error messages:
+<pre class="prettyprint devsite-code-highlight">
+'Payment app returned invalid response. Missing field "details".'
+'Payment app returned invalid response. Missing field "methodName".'
+</pre>
```
Comment (petele) [inline src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md:300]:
```suggestion

<pre class="prettyprint devsite-code-highlight">
'Payment app returned invalid response. Missing field "details".'
'Payment app returned invalid response. Missing field "methodName".'
</pre>
```

File: src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md [hunk 2]
```diff
@@ -289,6 +289,16 @@ choose a different payment app. The merchant website does not observe this, so
 there’s no need for detailed error codes to be sent from the payment app to the
 merchant website.
 
+Note: In Chrome 84 or later, if the activity result of a payment response
+received from the invoked payment app is set to `RESULT_OK`, then Chrome will
+check for non-empty `methodName` and `details` in its extras. If the validation
+fails Chrome will reject the `request.show()` promise with one of the following
+developer facing error messages:
```
Comment (petele) [inline src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md:296]:
```suggestion
Note: In Chrome 84 and later, if the activity result of a payment response
received from the invoked payment app is `RESULT_OK`, then Chrome will
check for non-empty `methodName` and `details` in its extras. If the validation
fails Chrome will return a rejected promise from `request.show()` with one of the following
developer facing error messages:
```

I think this is more clear, but I'm not 100% sure.

File: src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md (hunk 1 above)
Comment (petele) [reply src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md:300]: Just added an extra line break.

**Comment by sahel-sh** [comment]
LGTM

Thanks!

File: src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md (hunk 2 above)
Comment (agektmr) [reply src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md]: I think this is fine except the first `or`.

**Pull request merged**